endpoints/api/user.py

```diff
@@ -545,6 +546,78 @@ def delete(self):
         return "", 204
 
 
+@resource("/v1/user/initializexxxx")
```

Comment:
do we still need this endpoint? or is the endpoint defined in `web.py` sufficient?

Reply:
still trying to figure out how this method can be included in user.py instead of web.py. one or the other will "win"